### PR TITLE
Mention c-lightning adding multipath payments

### PR DESCRIPTION
**Pull request by laanwj**
Add a link to item mentioning the c-lightning PR to the list of mentions.

### CODE REVIEW

File: _topics/en/multipath-payments.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ primary_sources:
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: "C-Lightning #3809 adds support to C-Lightning for effective sending of multipath payments"
```
Comment (jonatack) [inline _topics/en/multipath-payments.md:61]: nits:

- maybe omit "to C-Lightning ", per the format of other entries

- unsure if "effective" if needed, though it is present in the first sentence of the linked-to news

Comment (jonatack) [reply _topics/en/multipath-payments.md:61]: Does the entry need to be at the bottom of the list (in the code, the display is sorted)?

**Comment by harding** [comment]
@laanwj Thanks for the PR!  We usually update the topic index on the first of each new month with links from the previous month, so the news from last week's newsletter would be added to the index in a few days, e.g. last month's update: https://github.com/bitcoinops/bitcoinops.github.io/pull/428  However, I have no objection to adding stuff early if people really want it.

FYI, this is failing because the title field is limited to 75 characters: https://travis-ci.org/github/bitcoinops/bitcoinops.github.io/builds/712578266#L673  I think you could just remove "to C-Lightning" from the title to get it to pass.

File: _topics/en/multipath-payments.md (hunk 1 above)
Comment (laanwj) [reply _topics/en/multipath-payments.md]: Thanks, updated for both

**Comment by laanwj** [comment]
OK, thanks, Seems fine to do it in one go later then. I don't know the process, just randomly noticed that this was missing.

**Comment by jonatack** [comment]
re-tested ACK

File: _topics/en/multipath-payments.md (hunk 1 above)
Comment (harding) [reply _topics/en/multipath-payments.md]: @jonatack FYI: entries can be anywhere in the YAML; the display logic sorts by date.

**Comment by harding** [comment]
Untested ACK.  Thanks!

**Comment by jnewbery** [comment]
Thanks  @laanwj ! You've just joined a very elite team of optech website contributors :astronaut:

**Pull request merged**